### PR TITLE
Add ctapipe recipe

### PR DESCRIPTION
**Pull request by maxnoe**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/ctapipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  imports:
+    - ctapipe
+    - ctapipe.core
+    - ctapipe.image
+    - ctapipe.calib
+    - ctapipe.instrument
+    - ctapipe.visualization
+    - ctapipe.io
+  commands:
+    - ctapipe-info --version
```
Comment (marcelotrevisani) [inline recipes/ctapipe/meta.yaml:67]: Can we try to add ``pip check`` here please?

```suggestion
    - ctapipe-info --version
    - pip check
  requires:
    - pip
```

**Comment by maxnoe** [comment]
@marcelotrevisani Thanks for the hint!

One question: one of the errors in `pip check` was `zstandard requires cffi but is not installed`. This means that the zstandard recipe is missing a requirement here, right? Normally I shouldn't need to declare transitive dependencies, correct?

**Comment by marcelotrevisani** [comment]
> @marcelotrevisani Thanks for the hint!
> 
> One question: one of the errors in `pip check` was `zstandard requires cffi but is not installed`. This means that the zstandard recipe is missing a requirement here, right? Normally I shouldn't need to declare transitive dependencies, correct?

If it is complaining about it, it indicates that you are probably missing that dependency

**Comment by maxnoe** [comment]
> If it is complaining about it, it indicates that you are probably missing that dependency

ctapipe depends on zstandard and that is listed.

Now pip check complains that zstandard depends on cffi but that cffi is not installed.

That for me means that the zstandard recipe is missing the cffi requirement.

**Comment by maxnoe** [comment]
And indeed, https://github.com/conda-forge/zstandard-feedstock/blob/master/recipe/meta.yaml

cffi is only listed for `host`, but it is also needed at runtime, right?

**Comment by marcelotrevisani** [comment]
> @marcelotrevisani Thanks for the hint!
> 
> One question: one of the errors in `pip check` was `zstandard requires cffi but is not installed`. This means that the zstandard recipe is missing a requirement here, right? Normally I shouldn't need to declare transitive dependencies, correct?

Yeap, it seems it is missing cffi on host

**Pull request merged**